__tests__/src/selectors/manifests.test.js

```diff
@@ -80,6 +80,12 @@ describe('getManifestoInstance', () => {
     const received = getManifestoInstance(state, { manifestId: 'x' });
     expect(received.id).toEqual('http://iiif.io/api/presentation/2.1/example/fixtures/19/manifest.json');
   });
+  it('is cached based off of input props', () => {
```

Comment:
I'm not sure how to better test this. This isn't a failing test to start with unfortunately, but I'm unsure how to actually test the recompution based off of different component instances calling our selector.